_posts/news/2020-02-19-announcing-scalajs-1.0.0.md

```diff
@@ -0,0 +1,581 @@
+---
+layout: post
+title: Announcing Scala.js 1.0.0
+category: news
+tags: [releases]
+permalink: /news/2020/02/19/announcing-scalajs-1.0.0/
+---
+
+
+We are thrilled to announce the General Availability release of Scala.js 1.0.0!
+
+After 7 years of development, including 5 years of stability within the 0.6.x series, we are finally ready to present Scala.js 1.0.0.
+Quoting Antoine de Saint-Exupéry,
+
+> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.
```

Comment:
👍 ❤️ Rigor and obsession in the Scala.js development process is an example for all of us emulate! Thank you!